lib/matplotlib/axes/_axes.py

```diff
@@ -6411,11 +6411,18 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
         """
         Plot a histogram.
 
-        Compute and draw the histogram of *x*.  The return value is a tuple
-        (*n*, *bins*, *patches*) or ([*n0*, *n1*, ...], *bins*, [*patches0*,
-        *patches1*, ...]) if the input contains multiple data.  See the
-        documentation of the *weights* parameter to draw a histogram of
-        already-binned data.
+        Computes the histogram of the data in *x* (see ~numpy.histogram) and
+        draws the results. If the input is an array, then the return value is
+        a tuple (*n*, *bins*, *patches*); if the input is a sequence of arrays,
+        then the return value is a tuple
+        ([*n0*, *n1*, ...], *bins*, [*patches0*, *patches1*, ...]).
```

Comment:
I think the whole multiple data discussion should be moved two paragrahs down to l.6427. And then we can merge the remaining sentence with the following paragraph.